.vscode/launch.json

```diff
@@ -2,14 +2,13 @@
     "version": "0.2.0",
     "configurations": [
         {
-            "name": ".NET Core Launch",
+            "name": "PowerShell Dev Console",
             "type": "coreclr",
             "request": "launch",
-            "justMyCode": false,
-            "stopAtEntry": true,
             "program": "${workspaceRoot}/debug/pwsh",
-            "preLaunchTask": "Build",
-            "externalConsole": true,
+            "console": "integratedTerminal",
+            "justMyCode": false,
+            "stopAtEntry": false,
```

Comment:
Is this only for C++?

Reply:
No, this makes it so that the PowerShell dev version starts in the vscode terminal pane rather than in an entirely separate window. This is necessary to make it work in codespaces/devcontainer.

Reply:
Do you say about `stopAtEntry`?

Reply:
For stopAtEntry, this causes a breakpoint to always happen at the start of the program. I found this annoying, as the PowerShell start point in Program Main is pretty discoverable if you want one, so I removed it.